hazelcast-sql/src/test/java/com/hazelcast/sql/impl/calcite/opt/OptimizerTestSupport.java

```diff
@@ -155,22 +155,24 @@ private static PhysicalRel optimizePhysicalInternal(
     }
 
     protected static HazelcastTable partitionedTable(
```

Comment:
I would propose to have two overloads, to avoid complication for non-HD tests (and the majority of tests are non-HD):
```
partitionedTable(name, fields, indexes, rowCount)
partitionedTable(name, fields, indexes, rowCount, nativeMemoryEnabled)
```